test/cmdlineTests/standard_yul_stack_opt/input.json

```diff
@@ -36,4 +42,24 @@
 	let a1, b1, c1, d1, e1, f1, g1, h1, i1, j1, k1, l1, m1, n1, o1, p1 := fun()
 	let a2, b2, c2, d2, e2, f2, g2, h2, i2, j2, k2, l2, m2, n2, o2, p2 := fun()
 	sstore(a1, a2)
+}"
+		}
+	},
+	"settings":
+	{
+		"optimizer": {
+			"enabled": false,
+			"details": {
+				"yul": true,
+				"yulDetails": {
+					"stackAllocation": true,
+					"optimizerSteps": "c"
```

Comment:
I could not disable the optimizer completely because then it produces an ICE even with stack optimization (`Variable a3 is 1 slot(s) too deep inside the stack`). Adding even this single step makes it compile.